src/crypto/BUILD.gn

```diff
@@ -59,9 +59,14 @@ buildconfig_header("crypto_buildconfig") {
   if (chip_with_se05x == 1) {
     defines += [ "CHIP_CRYPTO_HSM=1" ]
     defines += [ "CHIP_CRYPTO_HSM_NXP=1" ]
+  } else if (chip_enable_trustm == true) {
+    defines += [ "CHIP_CRYPTO_HSM=1" ]
+    defines += [ "CHIP_CRYPTO_HSM_NXP=0" ]
```

Comment:
Why is this line needed? If you need to force it zero, it means it's used too many places it should not be and we should fix it.

Reply:
Hi @tcarmelveilleux, Thanks for your review. We have fixed it in the latest commit: [Updated here](https://github.com/ying-css/connectedhomeip/blob/d8c4a946ae0e9d42c0c6935499e231054049001a/src/crypto/BUILD.gn#L64).